client_cli/pytests/models/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""
+This module provides classes for working with Iroha entities,
```

Comment:
```suggestion
Classes for interoperability with Iroha entities. Specifically: 
```

Reply:
When you call `help()` it already tells you that what you're looking at is a module. Listing classes specifically doesn't hurt.

Reply:
Created #3564